src/test/java/io/openliberty/tools/intellij/it/SingleModAppTestCommon.java

```diff
@@ -86,7 +86,97 @@ public void testDropDownStartAction() {
     }
 
     /**
-     * Prepares the the environment to run the tests.
+     * Tests Liberty Tools start/RunTests/stop actions run from the project's drop-down action menu.
+     */
+    @Test
+    public void testRunTestsActionUsingDropDownMenu() {
+        String testName = "testDropDownStartAction";
```

Comment:
update name for this test - maybe `testDropDownRunTests` ?

Reply:
Thanks Trevor, fixed this one and the other comments.